### PR TITLE
Edit to README

### PR DESCRIPTION
**Pull request by Nauman702**

### CODE REVIEW

**Comment by Nauman702** [comment]
Edited Readme.md

**Comment by Nauman702** [comment]
@nipraxis-fall-2022/instructors

File: README.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ Change directory back to root of repository
 cd ..
 ```
 
+## Installation
+```
+1. Install python dependencies with pip 
+
+```
```
Comment (matthew-brett) [inline README.md:36]:
```suggestion

1. Install python dependencies with pip 
```

**Review by matthew-brett — COMMENTED** [review]
Did I miss something else to review here?

File: README.md (hunk 1 above)
Comment (Nauman702) [reply README.md:36]: @matthew-brett No sorry, I was just testing the instructors' tag as one of the instructor told me to.

**Pull request closed without merging**